src/test/cypress/e2e/course/CourseExercise.cy.ts

```diff
@@ -4,11 +4,11 @@ import multipleChoiceQuizTemplate from '../../fixtures/quiz_exercise_fixtures/mu
 import { convertCourseAfterMultiPart } from '../../support/requests/CourseManagementRequests';
 import { generateUUID } from '../../support/utils';
 
-// Accounts
-const admin = artemis.users.getAdmin();
+// Users
+const users = artemis.users;
```

Comment:
`users` is just used below so maybe even cancel this const? I quickly searched for users but couldn't find it anywhere else then being used in the next const definition.

Reply:
Thanks for the hint!

In this file true, but in many other files we use users multiple times. One for each user we need, so something like 
```
const users = artemis.users;
const admin = users.getAdmin();
const tutor = users.getTutor();
const studentOne = users.getStudentOne();
...
```
So yes in this case the additional `const users` is not really necessary, but to stay consistent throughout the E2E tests, I would like to leave it here. The chances are also good, that we will extend this test file in the near future and add additional user.